### PR TITLE
Swagger for facility locator 

### PR DESCRIPTION
**Pull request by patrickvinograd**
Fixes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3813

WIP PR so I don't lose track of it.

### CODE REVIEW

**Comment by patrickvinograd** [comment]
This is ready for review.

File: spec/request/swagger_spec.rb [hunk 1]
```diff
@@ -749,6 +749,23 @@
...
+      context 'successful calls' do
+        it 'supports getting a list of facilities' do
+          VCR.use_cassette('facilities/va/pdx_bbox') do
+            expect(subject).to validate(:get, '/v0/facilities/va', 200,
+                                        'bbox' => ['-122.440689', '45.451913', '-122.78675', '45.64'])
+          end
+        end
+
+        it 'supports getting a list of facilities' do
+          VCR.use_cassette('facilities/va/vha_648A4') do
```
Comment (mphprogrammer) [inline spec/request/swagger_spec.rb]: What happens if the facility id does not exist?

File: app/swagger/requests/va_facilities.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          key :operationId, 'showFacility'
+          key :tags, %w(facilities)
+
+          parameter do
+            key :name, :id
+            key :in, :path
+            key :type, :string
+            key :required, true
+          end
+          response 200 do
```
Comment (mphprogrammer) [inline app/swagger/requests/va_facilities.rb]: What if the facility id does not exist?

File: app/swagger/requests/va_facilities.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            key :name, 'services[]'
+            key :in, :query
+            key :type, :array
+            key :collectionFormat, :multi
+            items do
+              key :type, :string
+            end
+          end
+
+          response 200 do
```
Comment (mphprogrammer) [inline app/swagger/requests/va_facilities.rb]: Is the bbox normalized, and if so, what if the box is degenerate (ex. [1, 1, 1, 1])?.

Comment (patrickvinograd) [inline app/swagger/requests/va_facilities.rb]: The GIS endpoint is pretty tolerant as long as the bounding box is numeric, it tries to make sense of it. But we will return a 422 for some cases where there are wrong number of params and things like that. Will add a response for that.

File: app/swagger/requests/va_facilities.rb (hunk 2 above)
Comment (patrickvinograd) [inline app/swagger/requests/va_facilities.rb]: Yep. Will add a response.

**Comment by patrickvinograd** [comment]
Added some error specs.

**Comment by patrickvinograd** [comment]
Fixed silly classname error, this is ready for a 👍 . I'm sure there is a corner case or two that could be addressed but this is a step in the right direction and I believe would document the API well enough for somebody to consume it.

**Pull request merged**